doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1023 @@
+11. Support Tunnel ECMP and underlay path failovers.
+12. Support a common VLAN-VNI map for all the EVPN tunnels.
+
+
+
+Following requirements will be attempted on a best effort basis. 
+
+1. Support monitoring and display of tunnel operational status.
+2. Support tunnel packet/octet  tx/rx counters.
+
+
+
+## 2.2 Configuration and Management Requirements
+
+Configuration and Management for this feature will be based the proposed management framework for SONiC.
+
+1. Support configuration of Source IP address of the VTEP. 
+2. Support configuration of a global VLAN-VNI map.
+3. Support configuration of L3VNI association with VRF. 
+
+
+
+FRR version 7.0.1, and later, is assumed here. FRR configuration is assumed in "split" configuration mode.
+BGP EVPN configurations in FRR are referred wherever required in this document. However, this document does not serve as reference guide for the FRR configurations.
+
+
+
+## 2.3 Scalability Requirements
+
+1. Total Local VXLAN terminations (source VTEP) - 1.
+2. Total Remote VTEPs (VXLAN destination tunnels) - 512.
+3. Total L2 VNI per switch- 4K.
+4. Total VNI per tunnel - 4K.
+5. Total EVPN participating VRF per switch - 512
+
+
+
+## 2.4 Warm Boot Requirements
+
+Warm reboot is intended to be supported for the following cases:
+
+- Planned system warm reboot. 
+- Planned SwSS warm reboot.
+- Planned FRR warm reboot.
+
+
+
+# 3 Feature Description
+
+## 3.1 Target Deployment use cases
+
+The following are some of the deployment use cases for L2 and L3 VXLAN. 
+
+- The Leaf nodes of the IP Fabric Leaf-Spine topology.
+- The Aggregation or Core or collapsed Core-Aggregation nodes of the traditional Access, Aggregator, and Core topology.
+- Multi tenant environments.
+- VLAN extension over an IP WAN. 
+- Datacenter Interconnect using L2 and L3 hand-off.
+
+
+
+## 3.2 Functional Description
+
+EVPN VXLAN feature supports L2 forwarding and L3 routing over VXLAN, dynamic creation and deletion of VXLAN tunnels, dynamic extension and removal of VLANs over the discovered tunnels based on BGP EVPN procedures described in RFC 7432 and 8365.
+
+#### L2 forwarding over VXLAN
+
+Traffic received from the access ports are L2 forwarded over VXLAN tunnels if the DMAC lookup result does not match the switch MAC. The frame format is as described in RFC 7348. The VNI is based on the configured global VLAN-VNI map. BUM traffic is ingress replicated to all the tunnels  which are part of the VLAN. The DIP of the BUM packets is the IP address of the remote VTEP. L2 unicast traffic is forwarded to the tunnel to which the MAC points to. 
+Traffic received from VXLAN tunnels are mapped to a VLAN based on the received VNI and the VNI-VLAN lookup. The received traffic can be L2 unicast or BUM depending on the inner MAC lookup. Traffic received from VXLAN tunnels are never forwarded onto another VXLAN tunnels. 
+
+![L2 forwarding over VXLAN ](images/pktflowL2.PNG "Figure 1 : L2 Flow")
+__Figure 1: L2 forwarding Use Case__
```

Comment:
The figure is not a use case diagram. May be rename it to packet flow.

Reply:
will do.